frontend/src/components/GddSend/TransactionForm.vue

```diff
@@ -188,6 +188,15 @@ export default {
   methods: {
     onSubmit() {
       this.normalizeAmount(true)
+      // remove emotjis from memo
+      this.form.memo = this.form.memo
+        .replace(
+          /([\u2700-\u27BF]|[\uE000-\uF8FF]|\uD83C[\uDC00-\uDFFF]|\uD83D[\uDC00-\uDFFF]|[\u2011-\u26FF]|\uD83E[\uDD10-\uDDFF])/g,
+          '',
+        )
+        .replace(/\s+/g, ' ')
+        .trim()
```

Comment:
There comming some questions into my mind:

- why emojis do not work in MariaDB?
- do other UTF-8 chars work in MariaDB?
- we should do it in the backend as well! because of future APIs …
- in case this has to be filtered the user should see that this they can not type it